### DISCUSSION TITLE
Support for TrackballControls-like functionality

### DISCUSSION THREAD

**Discussion by MattFerraro** [yomotsu/camera-controls#472] [Ideas]
I love all the extra functionality this library provides, but sometimes I need something more similar to [TrackballControls](https://threejs.org/docs/#examples/en/controls/TrackballControls). On its face this should be as simple as not keeping a constant camera up vector, but I'm sure that's easier said than done.

Still, is there any way this could be supported inside camera-controls?

**Comment by yomotsu** [comment]
Actually, TrackballControls is not compatible with OrbitControls.
camera-controls is a sort of OrbitControls and we can't add TrackballControls features.
